packages/neo-one-client/src/__tests__/user/LocalUserAccountProvider.test.ts

```diff
@@ -1076,4 +1076,60 @@ describe('LocalUserAccountProvider', () => {
     expect(result2).toHaveProperty('transaction');
     verifyMocks();
   });
+
+  test('transaction consolidates', async () => {
+    const unspent = [outputs.oneNEO, outputs.twoNEO, outputs.sevenNEO, outputs.oneGAS, outputs.elevenGAS];
+    const transfer = [
+      {
+        amount: new BigNumber('3'),
+        asset: common.NEO_ASSET_HASH,
+        to: 'AVf4UGKevVrMR1j3UkPsuoYKSC4ocoAkKx',
+      },
+    ];
+    keystore.byteLimit = 800;
+    keystore.sign = jest.fn(() => witness);
+    provider.getUnspentOutputs = jest.fn(() => Promise.resolve(unspent));
+    provider.getTransactionReceipt = jest.fn(() => Promise.resolve('receiptDummy'));
+
+    const spy = jest.spyOn(keystore, 'sign');
+    await localUserAccountProvider.transfer(transfer, {
+      from: options.from,
+      attributes: options.attributes,
+      networkFee: undefined,
+    });
+
+    expect(spy.mock.calls[0][0].message.length).toEqual(732);
```

Comment:
This still doesn't really show what's being verified. Sure, the size is less than 800, but don't we expect more inputs and outputs? You should be verifying that the transaction has the inputs and outputs you expect.